proto/v1/versions/api_versions.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.versions;
+
+import "google/protobuf/descriptor.proto";
+
+option java_package = "org.oppia.proto.v1.versions";
+option java_multiple_files = true;
+
```

Comment:
In these and other files is it reasonable to have a "Fileoverview" explaining what this file is about and what sorts of things should go in it?